test/rest-adapter.test.js

```diff
@@ -17,6 +17,10 @@ var factory = require('./helpers/shared-objects-factory.js');
 describe('RestAdapter', function() {
   var remotes;
 
+  before(function() {
+    process.env.NO_DEPRECATION = 'strong-remoting';
+  });
+
```

Comment:
You also need an `after` handler to reset the environment (remove the listener), so that deprecations are printed again.